pkgs/by-name/ne/nero-umu/package.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  qt6,
+  qt6Packages,
+  icu,
+  icoextract,
+  icoutils,
+  umu-launcher,
+  winetricks,
+  curl,
+}:
+stdenv.mkDerivation rec {
+  pname = "nero-umu";
+  version = "v1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "SeongGino";
+    repo = "Nero-umu";
+    tag = "${version}";
+    hash = "sha256-sX/Z/b5stauut8qg6IV/DdsCIkdx1N3+y1jwoXHr1LY=";
+  };
+
+  #Replace quazip git submodule with pre-packaged quazip
+  postUnpack = ''
+    rmdir source/lib/quazip/
+    ln -s ${qt6Packages.quazip.src} source/lib/quazip
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    qt6.wrapQtAppsHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qttools
+    qt6.qt5compat
+    qt6Packages.quazip
+    icu
+    icoextract
+    icoutils
+    umu-launcher
+    winetricks
+    curl
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DNERO_QT_VERSION=Qt6"
+  ];
+
+  #Fixes to be able to find binaries for nix
+  patches = [
+    ./nerofs.patch
+    ./neroprefix.patch
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 "nero-umu" "$out/bin/nero-umu"
+    for size in 32 48 64 128; do
+      install -Dm644 "${src}/img/ico/ico_"$size".png" "$out/share/icons/hicolor/"$size"x"$size"/apps/xyz.TOS.Nero.png"
+    done
+    install -Dm644 "${src}/xyz.TOS.Nero.desktop" "$out/share/applications/xyz.TOS.Nero.desktop"
+    runHook postInstall
+  '';
+
+  preFixup = ''
+    qtWrapperArgs+=(
+      --prefix PATH : $out/bin:${
+        lib.makeBinPath [
+          icoextract
+          icoutils
+          winetricks
+        ]
+      }
+    )
+  '';
+
+  meta = {
+    homepage = "https://github.com/SeongGino/Nero-umu";
+    description = "Fast and efficient Proton prefix runner and manager using umu as backend";
+    license = lib.licenses.gpl3Plus;
+    mainProgram = "nero-umu";
+    platforms = [ "x86_64-linux" ];
+    maintainers = with lib.maintainers; [
+      ern775
+      blghnks
```

```suggestion
      blghnks
      keenanweaver
```

I would like to be a maintainer. :smile: